data/migrations/20200121133810_create-cars-table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("cars", table => {
+    table.increments();
+    table
+      .string("vin", 17)
+      .unique()
+      .notNullable();
+    table.text("make").notNullable();
+    table.text("model").notNullable();
+    table.integer("mileage").notNullable();
+    table.text("transmissionType");
+    table.text("titleStatus");
```

Comment:
Ensure you use appropriate data types for fields. If the amount of text expected of a field is within the range of 255 characters, you probably need to use the string method as you did for vin